codefresh/pipeline-runner/entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -x
+set -e -o pipefail
 
 codefresh auth create-context context --api-key "$CF_API_KEY"
```

Comment:
Codefresh cli autoamtically looks for `CF_API_KEY`
```suggestion

codefresh auth create-context context
```

Reply:
How is this script used?